c++/include/orc/Statistics.hh

```diff
@@ -37,12 +37,76 @@ namespace orc {
      * of rows because of NULL values and repeated values.
      * @return the number of values
      */
-    virtual uint64_t getNumberOfValues() const = 0;
+    virtual uint64_t getNumberOfValues() const {
+      return valueCount;
+    }
+
+    /**
+     * Set the number of values in this column
+     * @param newValueCount new number of values to be set
+     */
+    virtual void setNumberOfValues(uint64_t newValueCount) {
```

Comment:
Are these and the below methods required in the public API?
I think they belong to the Impl classes in `c++/src/Statistics.hh`.